src/common/Button/Button.tsx

```diff
@@ -9,8 +9,9 @@ type ButtonProps = {
     | 'medium'
     | 'large'
     | 'super-large'
-    | 'extra-large';
-  label: string;
+    | 'extra-large'
+    | 'costum-btn';
```

Comment:
```suggestion
    | 'custom-btn';
```

Search and replace everywhere.

Reply:
changed